components/src/dynamo/common/tests/multimodal/test_image_loader.py

```
@@ -37,6 +40,33 @@
+
+        from dynamo.common.http.url_validator import validate_media_url
+        from dynamo.common.multimodal import ImageLoader
+
+        assert ImageLoader.__name__ == "ImageLoader"
+        assert callable(validate_media_url)
+        assert "dynamo.common.multimodal.embedding_transfer" not in sys.modules
+        assert "dynamo.common.memory.multimodal_embedding_cache_manager" not in sys.modules
+        """
+    )
+
+    subprocess.run([sys.executable, "-c", script], check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)